spec-0005/index.md

```diff
@@ -25,6 +26,35 @@ intended use-cases, and pseudo-code illustrating its use.
 Discuss how this would be implemented.
 -->
 
+### Build nightly wheels
+
+By building and uploading wheel for your project you make it easier for projects
+that depend on you to test against your latest changes. They can then report problems
+they find.
+
+There are a few steps to implementing this for your project:
+1. Get access to the area that nightly wheels are uploaded to
+2. Setup a CI step that builds wheels for your project
+3. Setup a CI step that uploads wheels to https://anaconda.org/scientific-python-nightly-wheels/
+
+For step (1) contact XXX. Deposit the token you get as a [secret on your repository](https://docs.github.com/en/actions/security-guides/encrypted-secrets) named `UPLOAD_TOKEN`.
```

Comment:
Put a placeholder so #168 can be mentioned here when it is accepted?

Reply:
I don't know what is best here. In some way there is no need for this token to be stored in a readable way. If for some reason you need the value of the token you can have it regenerated. However, people might not realise that the GitHub Secrets are read-only?